frontend/providers/applaunchpad/public/locales/en/translation.json

```diff
@@ -0,0 +1,123 @@
+{
+  "Applications": "Applications",
+  "New Application": "New Application",
+  "Cancel": "Cancel",
+  "Confirm": "Confirm",
+  "Name": "Name",
+  "Status": "Status",
+  "Creation Time": "Creation Time",
+  "CPU": "CPU",
+  "Memory": "Memory",
+  "Number of Instances": "Number of Instances",
+  "Storage Capacity": "Storage Capacity",
+  "Operation": "Operation",
+  "Running": "Running",
+  "Paused": "Paused",
+  "Active": "Active",
+  "Details": "Details",
+  "Start Up": "Start Up",
+  "Delete": "Delete",
+  "Pause": "Pause",
+  "Restart": "Restart",
+  "Change": "Change",
+  "ApplicationType": "ApplicationType",
+  "Basic Information": "Basic Information",
+  "Deployment Mode": "Deployment Mode",
+  "Advanced Configuration": "Advanced Configuration",
+  "Start Command": "Start Command",
+  "Run Parameters": "Run Parameters",
+  "Not Configured": "Not Configured",
+  "Accessible to the Public": "Accessible to the Public",
+  "Configuration File": "Configuration File",
+  "Storage Volume": "Storage Volume",
+  "Environment Variables": "Environment Variables",
+  "Fixed Number of Instances": "Fixed Number of Instances",
+  "Image Name": "Image Name",
+  "Private": "Private",
+  "Public": "Public",
+  "Image Name (Private)": "Image Name (Private)",
+  "Real-time Monitoring": "Real-time Monitoring",
+  "Network Configuration": "Network Configuration",
+  "Intranet Address": "Intranet Address",
+  "Internet Address": "Internet Address",
```

Comment:
duplicated

Reply:
One internal network address and one external network address, I changed it to External Address